src/commands/snapshot.ts

```diff
@@ -0,0 +1,105 @@
+import {flags} from '@oclif/command'
+import Constants from '../services/constants'
+import {StaticSnapshotOptions} from '../services/static-snapshot-options'
+import StaticSnapshotService from '../services/static-snapshot-service'
+import PercyCommand from './percy-command'
+
+export default class Snapshot extends PercyCommand {
+  static description = 'Snapshot a directory of static site assets'
+  static hidden = true
+
+  static args = [{
+    name: 'staticAssets',
+    description: 'The path to the compiled static asset directory.',
+    required: true,
+  }]
+
+  static examples = [
+    '$ percy snapshot _mySite/',
+    '$ percy snapshot _mySite/ --baseUrl "blog/"',
+    '$ percy snapshot _mySite/ --widths "320,780,1280"',
+    '$ percy snapshot _mySite/ --ignore-folders "Tmp,_secrets,node_modules"',
+  ]
+
+  static flags = {
+    'snapshot-capture-regex': flags.string({
+      char: 'c',
+      description: 'Regular expression for matching the files to snapshot. Defaults to: "\.(html|htm)$"',
+      default: '\.(html|htm)$',
+    }),
+    'ignore-folders': flags.string({
+      char: 'i',
+      description: 'Comma-seperated string of folders to ignore. Ex: Tmp,_secrets,node_modules',
+      default: '',
+    }),
+    'widths': flags.string({
+      char: 'w',
+      description: 'Comma-separated string of rendering widths for snapshots. Ex: 320,1280',
+      default: '1280',
```

Comment:
Should we rely on the defaults that are in the API? I remember seeing something that removed the defaults from agent in favor of that

Reply:
yep, I think we should remove the `--widths` flag completely. You will be able to use the `.percy.yml` file to set them if you want to.

Reply:
👍